pages/query.js

```diff
@@ -128,6 +130,35 @@ export default function Query() {
       <Navigation />
       <Breadcrumbs />
       <PagePreamble />
+      <label className="m-2 relative inline-flex cursor-pointer select-none items-center">
+        <input
+          type="checkbox"
+          checked={!isGrch38}
+          onChange={(e) => {
+            setIsGrch38(!e.target.checked);
+          }}
+          className="sr-only"
```

Comment:
A checkbox is only apparent to screen readers, but invisible to sighted users?

Reply:
This caught my eye as well. But since I borrowed the design from here: https://tailgrids.com/components/toggle-switch, I decided just keep it.

Reply:
Oh, I see. I didn’t realize this was for a switch that uses the standard browser checkbox mechanism. Yeah, that’s the right way to do it as far as I’m aware. In the future, might extract this into its own component so that you can use it anywhere, and so you can document it so the next person doesn’t wonder why that’s there.

Reply:
Sounds good!